packages/snap-preact/src/setupEvents.test.tsx

```diff
@@ -193,4 +207,67 @@ describe('setupEvents', () => {
 			jest.clearAllMocks();
 		});
 	});
+
+	describe('controller/updateRecs', () => {
+		it('can listen for controller/updateRecs event and invoked without parameters', () => {
+			expect(makeSearchSpy).not.toHaveBeenCalled();
+
+			//no data passed at all
+			expect(() => {
+				eventManager.fire('controller/updateRecs');
+			}).not.toThrow();
+
+			expect(makeSearchSpy).toHaveBeenCalledTimes(1);
```

Comment:
We should test that multiple controllers search function was called (would need multiple spies perhaps?).